src/cli/cli.js

```diff
@@ -25,8 +26,13 @@ export default function cli(args: string[]) {
     .command(
       'bump',
       'bumps "next" changelog to new version',
-      () => {},
-      withAPI((changelog) => runBump(changelog))
+      (yargs) => {
+        yargs.option('version', {
+          alias: 'v',
+          describe: 'next version to bump to'
+        });
+      },
+      withAPI((changelog, argv: CLIBumpOptionsType) => runBump(changelog, argv))
```

Comment:
Is there possibly a need to validate the given `argv` in here? or within `runBump()`?

Reply:
Or maybe you could adjust the description to hint the user to a proper version format. smthg. like: `next version to bump to (e.g. 1.0.1)`

Reply:
@slayerpart That is intentionally not done since `--version` allows for non-SemVer-compliant versions
Since we cannot know what this means, we trust the version to be "valid"

Reply:
Fair enough! :)